app/routes/router.js

```diff
@@ -92,6 +92,16 @@ router
     .get('/findalltransactionsbyparamid', function (req, res){
         API.findTransactionsByParamId(req, res)
     })
+	
+	// gets all the errors
+    .get('/findallerrorreports', function (req, res){
```

Comment:
@brnsmns13  This seems like a code clarity issue, if It really bothers you we can change it, but at this point I'm more worried about functionality. Ideally, you'd be using the README.MD anyways - though we do need to update this to clarify some information.